### ISSUE TITLE
asadmin login fails when using interactively

### ISSUE THREAD

**Issue by eduarddrenth** [payara/Payara#3544]
# Description #
----------

Cannot login, asadmin seems to immediately send <enter>

## Expected Outcome

should be able to login using asadmin like in 5.183:

```
eduard@eduard-X756UB:~$ ./payara5183/bin/asadmin
Use "exit" to exit and "help" for online help.
asadmin> login
Enter admin user name [Enter to accept default]> admin
Enter admin password> 
Admin login information for host [localhost] and port [4848] is being overwritten with credentials provided. This is because the --savelogin option was used during create-domain command.
Login information relevant to admin user name [admin] for host [localhost] and admin port [4848] stored at [/home/eduard/.gfclient/pass] successfully.
Make sure that this file remains protected. Information stored in this file will be used by administration commands to manage associated domain.
Command login executed successfully.
asadmin> quit
Command multimode executed successfully.
```

## Current Outcome

when trying to login all keys seem to be translated to <enter>:

eduard@eduard-X756UB:~$ ./payara5184/bin/asadmin
Use "exit" to exit and "help" for online help.
asadmin> login
Enter admin user name [Enter to accept default]> Enter admin password> 
Authorization has been refused for credentials [user: a] given in this request.
(Usually, this means invalid user name and/or password)
Command login failed.
asadmin> mmiinn

## Steps to reproduce (Only for bug reports) 

- unzip payara 184
- ./payara5/bin/asadmin
- login
- start typing

NOTE: when using "asadmin login" logging in succeeds

## Environment ##

- **Payara Version**: 5.184
- **Edition**: Full
- **JDK Version**: openjdk version "1.8.0_191"
- **Operating System**: ubuntu 1804

**Comment by Cousjava** [comment]
This is a known issue PAYARA-3357 that has a PR to fix it at #3528

**Comment by eduarddrenth** [comment]
great....did not find that issue because it did not mention asadmin nor login
